### ISSUE TITLE
Error in sorting data-sets based on title

### ISSUE THREAD

**Issue by rushirajnenuji** [NCEAS/metacatui#350]
The datasets when sorted based on the title(a-z) results into sorting based on the upper case and lower case separately. 
![sorting error](https://user-images.githubusercontent.com/19696935/32521340-f83f9072-c3c7-11e7-9d1e-601888dbf2d8.png)

**Comment by laurenwalker** [comment]
This is the sorting done by Solr, so isn't something we can fix from MetacatUI.

**Comment by csjx** [comment]
Hey @rushirajnenuji - thanks for noticing this. As @laurenwalker pointed out, this can't be controlled from MetacatUI, so it may be worth looking at the Solr 5 documentation to see if this is a configurable feature in the `solr-config.xml` file that Metacat deploys. If this is something we can adjust, add a ticket in over at https://projects.ecoinformatics.org/ in the Metacat section, and we'll try to address it.

**Comment by rushirajnenuji** [comment]
Hey @csjx, I've found one solution on this [StackOverflow site](https://stackoverflow.com/questions/2053214/how-to-create-a-case-insensitive-copy-of-a-string-field-in-solr)

This is fixable issue from the solr-config.xml. Turns out for `class="solr.StrField"` does not analyze the string while `class="solr.TextField"` does. I believe this should fix the issue. I'll add the ticket at given URL. Thanks!
